notifications/src/main/java/org/openedx/notifications/presentation/settings/NotificationsSettingsFragment.kt

```diff
@@ -0,0 +1,233 @@
+                )
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+private fun NotificationsSettingsScreen(
+    windowSize: WindowSize,
+    notificationsConfiguration: NotificationsConfiguration,
+    discussionPreferenceChanged: (Boolean) -> Unit,
+    onBackClick: () -> Unit,
+) {
+    val scaffoldState = rememberScaffoldState()
+
+    Scaffold(
+        modifier = Modifier
+            .fillMaxSize()
+            .semantics {
+                testTagsAsResourceId = true
+            },
+        scaffoldState = scaffoldState
+    ) { paddingValues ->
+
+        val contentWidth by remember(key1 = windowSize) {
+            mutableStateOf(
+                windowSize.windowSizeValue(
+                    expanded = Modifier.widthIn(Dp.Unspecified, 420.dp),
+                    compact = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 24.dp)
+                )
+            )
+        }
+
+        val topBarWidth by remember(key1 = windowSize) {
+            mutableStateOf(
+                windowSize.windowSizeValue(
+                    expanded = Modifier.widthIn(Dp.Unspecified, 560.dp),
+                    compact = Modifier
+                        .fillMaxWidth()
+                )
+            )
+        }
+
+        Box(
+            modifier = Modifier.fillMaxSize(),
+            contentAlignment = Alignment.TopCenter
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(paddingValues)
+                    .settingsHeaderBackground()
+                    .statusBarsInset(),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Toolbar(
+                    modifier = topBarWidth
+                        .displayCutoutForLandscape(),
+                    label = stringResource(id = R.string.notification_push_notifications),
+                    canShowBackBtn = true,
+                    labelTint = MaterialTheme.appColors.settingsTitleContent,
+                    iconTint = MaterialTheme.appColors.settingsTitleContent,
+                    onBackClick = onBackClick
+                )
+
+                Box(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .clip(MaterialTheme.appShapes.screenBackgroundShape)
+                        .background(MaterialTheme.appColors.background)
+                        .displayCutoutForLandscape(),
+                    contentAlignment = Alignment.TopCenter
+                ) {
+                    Column(
+                        modifier = contentWidth
+                    ) {
+                        Row(
+                            Modifier
+                                .testTag("btn_discussions_activity")
+                                .fillMaxWidth()
+                                .height(92.dp)
```

Comment:
instead of fixing the height better to apply vertical padding for dynamically adjustment.